tests/functional/tests/login.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+var http = require('http');
+
+var LoginPage = require('../pages/LoginPage.js');
+var DesktopPage = require('../pages/DesktopPage.js');
+
+describe('Luceo login page', function () {
+    var loginPage;
+
+    beforeEach(function (done) {
+        loginPage = new LoginPage();
+        loginPage.get().then(done);
+    });
+
+    it('should correctly populate the dataLayer object', function () {
+        loginPage.getDataLayer().then(function (dataLayer) {
+            expect(dataLayer).toBeDefined();
+            var obj = dataLayer[0];
+            expect(obj["EEOUS"]).toBeTruthy();
+            expect(obj["Luceo Server Address"]).toBeDefined();
+            expect(obj["OFCCP"]).toBeFalsy();
+            //using contains instead of comparison
+            //to handle issue that port number shows up in circleci
+            expect(obj["SiteId"].indexOf("luceo-ocb-test.dev") > -1).toBeTruthy();
+            expect(obj["TalentNetworkFO"]).toBeTruthy();
```

Comment:
Mixed double and single quotes.<br>['TalentNetworkFO'] is better written in dot notation.